src/rpc/cli.cpp

```diff
@@ -87,9 +100,19 @@ void cli::run()
          {
             break;
          }
-         std::cout << line << "\n";
```

Comment:
I've always wondered why cli_wallet prints the input line. Can we remove that completely?

Reply:
I think we can remove it really completely if there aren't any reasons to have it.

Reply:
I 95% agree with this removal, just trying to find some use case of having it but cant find any valid. The only remotely i can think of is someone parsing cli wallet output and regex failing after this change.

By what i see the editline does not allow any environment variable or scripting, the repeat of the line could make sense if something like this were possible:

new>> get_account $MYVAR
get_account oxarbitrage
...

However this is not the case.

Reply:
Thanks @oxarbitrage ! I'm thinking about other use cases too :) still cannot find ...